test/snapshots/blueprint.test.ts.md

```diff
@@ -427,43 +433,45 @@ Generated by [AVA](https://avajs.dev).
                 {
                   code: {
                     go: {
-                      request: 'client.Foos.List(context.Background())',
+                      request: `import api "github.com/seamapi/go"␊
+                      ␊
+                        client.Foos.List(context.Background(), api.FoosListRequest(ObjectProp: api.FoosListRequestObjectProp{Foo: api.String("bar")}))`,
                       request_syntax: 'go',
                       response: '[{"foo_id":"8d7e0b3a-b889-49a7-9164-4b71a0506a33","name":"Best foo","nullable_prop":null,"number_prop":10,"object_prop":{"foo":"bar"},"array_prop":["foo","bar"]}]',
                       response_syntax: 'json',
                       title: 'Go',
                     },
                     javascript: {
-                      request: 'await seam.foos.list()',
+                      request: 'await seam.foos.list({"object_prop":{"foo":"bar"}})',
                       request_syntax: 'javascript',
                       response: '[{"foo_id":"8d7e0b3a-b889-49a7-9164-4b71a0506a33","name":"Best foo","nullable_prop":null,"number_prop":10,"object_prop":{"foo":"bar"},"array_prop":["foo","bar"]}]',
                       response_syntax: 'json',
                       title: 'JavaScript',
                     },
                     php: {
                       request: `<?php␊
-                      $seam->foos->list()`,
+                      $seam->foos->list(object_prop:{"foo":"bar"})`,
```

Comment:
It this correct? I thought PHP uses `=>`.

Reply:
Yea, seems like a bug. I'll take a look

Reply:
Opened a [PR](https://github.com/seamapi/blueprint/pull/106) into the current one fixing this php bug.